cms/templates/course_outline.html

```diff
@@ -69,6 +69,11 @@ <h3 class="sr">${_("Page Actions")}</h3>
                     </a>
                 </li>
                 <li class="nav-item">
+                    <a href="#" class="button re-index" data-category="reindex" title="${_('Reindex current course')}">
```

Comment:
Our naming convention would be to call this ``button-reindex``, not ``re-index``.